src/js/components/Popup/PopupTooltip.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+
+const propTypes = {
+  text: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
+  popupStyle: PropTypes.object,
+  testId: PropTypes.string,
+};
+
+type Props = React.FunctionComponent<PropTypes.InferProps<typeof propTypes>>;
+
+const PopupTooltip: Props = ({ text, testId, children }) => {
+  const buttonRef = React.useRef<HTMLButtonElement | null>(null);
+  const tooltipRef = React.useRef<HTMLDivElement | null>(null);
+  const [pos, setPos] = React.useState<{ top: number; left: number }>({ top: 0, left: 0 });
```

Comment:
I don't think we need to set the type in this "pos" state, it seems redundant for me.
Another thing, could you change it to "position"? It's more readable than just "pos"